doc/man3/SSL_CONF_cmd.pod

```diff
@@ -460,7 +473,7 @@ B<SessionTicket>: session ticket support, enabled by default. Inverse of
 B<SSL_OP_NO_TICKET>: that is B<-SessionTicket> is the same as setting
 B<SSL_OP_NO_TICKET>.
 
-B<Compression>: SSL/TLS compression support, enabled by default. Inverse
+B<Compression>: SSL/TLS compression support, disabled by default. Inverse
```

Comment:
If i remember well, TLS Compression is disabled by default, for security reasons.
So this is a documentation fix that should be backported ?

Reply:
Yup. Since I was in here dealing with (other) compression, I fixed it. But this would be appropriate for backporting.